integration_tests/log4rs/wallet.yml

```diff
@@ -93,6 +93,11 @@ loggers:
     appenders:
       - network
     additive: false
+  contacts:
```

Comment:
We need to add this to the default log of the wallet as well. 
But I would also use unique appended and not `- network`